.gitlab-ci.yml

```diff
@@ -654,10 +654,31 @@ library:ci-argosy:
 library:ci-bbv:
   extends: .ci-template
 
+library:ci-coqutil:
+  extends: .ci-template
```

Comment:
You need to use the same template for dependencies and dependent jobs, i.e. the `ci-template-flambda` one in this case.

Reply:
Do I also need the `needs:` field even if it `extends:`?